.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: 'BUG REPORT: '
+labels: ':type: bug'
+assignees: ''
+
+---
+
+*NOTE: This issue template is being created and may not be complete.*
+
+*NOTE: Please only submit bug reports to this repository for issues with the
+contents of the repository itself. General bug reports for Rocky Linux should
+be submitted to the [Rocky Linux Bug Tracker](https://bugs.rockylinux.org).*
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+
+- OS: [e.g. iOS]
+- Browser [e.g. chrome, safari]
+- Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+
+- Device: [e.g. iPhone6]
+- OS: [e.g. iOS8.1]
+- Browser [e.g. stock browser, safari]
+- Version [e.g. 22]
```

Comment:
Do we really need to capture this from a smartphone? It seems running Rocky on a smartphone would fall more under a SIG then general use.

Reply:
Nope... it's a copy-and-paste sample to generate discussion... success. I think more appropriate here might be a template for Server rather than Smartphone. The way the template is completed the reporter is able to remove / add as desired. We may want / need to be explicit about that though.